hyperspy/io_plugins/edax.py

```diff
@@ -921,6 +921,12 @@ def spd_reader(filename,
                           'reading .ipr {}'.format(ipr_fname))
             ipr_header = __get_ipr_header(f, endianess)
             original_metadata['ipr_header'] = sarray2dict(ipr_header)
+
+            # Workaround for type error when saving hdf5:
+            # save as list of strings instead of numpy unicode array
+            original_metadata['ipr_header']['charText'] = \
```

Comment:
For future reference, it may be good to add the links to the PR and https://github.com/h5py/h5py/issues/289.